src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -199,6 +204,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    handleActiveAnnotation = (feedItems: FeedItems): void => {
+        const { emitAnnotatorActiveChangeEvent, file, match, onVersionChange } = this.props;
+        const { annotatorState: { activeAnnotationId } = {} } = this.state;
```

Comment:
Should `annotatorState` default to an empty object to avoid needing this default parameter?